### PR TITLE
1st Commit - <3

### PR DESCRIPTION
**Pull request by Ichikuro**
Removed Spaces - Per Mergu

### CODE REVIEW

File: bomb.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    type: item
+    material: i@tnt
+    display name: Bomb
+
+bomb_entity:
+    type: entity
+    entity_type: e@zombie
+    invulnerable: true
+    custom_name: Bomb
+	age: baby
```
Comment (mergu) [inline bomb.yml]: fix indent

File: dungeondoor.yml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            }
+            else {
+                - define loc %4%
+                }
+        - if <el@%5%.is[==].to[null]> {
+			- define filter m@air
+			}
+			else {
+				- define filter %5%
+				}
```
Comment (mergu) [inline dungeondoor.yml]: what was caff thinking with these indents

File: dungeondoor.yml [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        - choose %3%:
+            - case "+x":
+                - define height <math:<%tl%.x.abs.as_int>-<%br%.x.abs.as_int>>
+                - if <el@%height%.is[less].than[0]> {
+                    - define height <el@%height%.abs>
+                    }
+                - define length <math:<%tl%.z.abs.as_int>-<%br%.z.abs.as_int>>
+				- define length <el@%length%.abs>
+                - define depth <math:<%tl%.y.abs.as_int>-<%br%.y.abs.as_int>>
+				- define depth <el@%depth%.abs>
```
Comment (mergu) [inline dungeondoor.yml]: gross

File: dungeondoor.yml [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+                    }
+            - case "-x":
+                - define height <math:<%tl%.x.abs.as_int>-<%br%.x.abs.as_int>>
+                - if <el@%height%.is[less].than[0]> {
+                    - define height <el@%height%.abs>
+                    }
+                - define length <math:<%tl%.z.abs.as_int>-<%br%.z.abs.as_int>>
+				- define length <el@%length%.abs>
+                - define depth <math:<%tl%.y.abs.as_int>-<%br%.y.abs.as_int>>
+				- define depth <el@%depth%.abs>
```
Comment (mergu) [inline dungeondoor.yml]: ew

File: dungeondoor.yml [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+                    }
+            - case "+z":
+                - define height <math:<%tl%.z.abs.as_int>-<%br%.z.abs.as_int>>
+                - if <el@%height%.is[less].than[0]> {
+                    - define height <el@%height%.abs>
+                    }
+                - define length <math:<%tl%.y.abs.as_int>-<%br%.y.abs.as_int>>
+				- define length <el@%length%.abs>
+                - define depth <math:<%tl%.x.abs.as_int>-<%br%.x.abs.as_int>>
+				- define depth <el@%depth%.abs>
```
Comment (mergu) [inline dungeondoor.yml]: ew

File: dungeondoor.yml [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+				- playsound <player> BLOCK_IRON_DOOR_CLOSE pitch:2
+				- actionbar "<&7> It's locked.." targets:<player>
+				}
+small_key:
+    type: item
+    debug: false
+    material: i@tripwire_hook
+    display name: <&2><&l>Small Key
+    lore:
+    - "ss"
```
Comment (mergu) [inline dungeondoor.yml]: indentation through this whole file needs some work

Comment (Ichikuro) [reply dungeondoor.yml]: Can i pretend this whole script doesn't exist?

**Pull request merged**